gulpfile.js

```diff
@@ -84,6 +84,11 @@ gulp.task('ps1tests', ['compileTests'], function (cb) {
         .pipe(gulp.dest(_testRoot));
 });
 
+gulp.task('copyTestData', ['compileTests'], function (cb) {
+    return gulp.src(['Tests/**/data/**'],{ dot: true })
+        .pipe(gulp.dest(_testRoot));
+});
```

Comment:
@ericsciple I renamed the directory from "resources" to "data" as advised. I also renamed the task to copyTestData so that it's clearer what it does.